### ISSUE TITLE
UTF-8 support

### ISSUE THREAD

**Issue by pan-mroku** [sersorrel/edmcoverlay2#6]
The problem: ED Recon outputs a few UTF characters. Overlay prints a bunch of garbage then 
![2023-01-15_23:20:04](https://user-images.githubusercontent.com/1093597/212570399-4dfc7b43-b06b-466d-9ffe-e67589bbede6.png)

I tried changing XDrawString to [Xutf8DrawString](https://linux.die.net/man/3/xutf8drawstring) but it seems that then I was not able to find a font on my system that would support that. Maybe that could be easily fixed by just finding the right font.
I also found that [there is another way](https://www.x.org/wiki/guide/fonts/#index2h2) of using fonts but I did not dig deeper.

Do you have some pointers how could this be done or should I dig in libXft library?

**Comment by sersorrel** [comment]
yup, the way it works at the moment is not ideal. as far as I remember, the solution to this is ultimately "stop using Xlib to render text" (and probably "stop using Xlib", honestly); using something like Pango for text rendering has always been on the list of things I wanted to do, probably as part of doing all rendering with cairo rather than directly through Xlib. unfortunately I never got around to that; I started rewriting edmcoverlay2 in Rust (https://github.com/sersorrel/edmcoverlay.rs), because I do not know C++ well enough to improve this version much more, but I don't remember how far I got.

for what it's worth: I haven't played Elite in months, and don't see myself starting to do so again any time soon, so my interest in continuing to develop this project is limited (but if you wanna try, then go ahead!)
